docs/devops-guide/docker.md

```diff
@@ -195,7 +223,7 @@ Variable | Description | Example
 `HTTP_PORT` | Exposed port for HTTP traffic | 8000
 `HTTPS_PORT` | Exposed port for HTTPS traffic | 8443
 `JVB_ADVERTISE_IPS` | IP addresses of the Docker host (comma separated), needed for LAN environments | 192.168.1.1
-`PUBLIC_URL` | Public URL for the web service | https://meet.example.com
+`PUBLIC_URL` | Public URL for the web service | <https://meet.example.com>
```

Comment:
Don't put the angle brackets or people will think they are necessary.

Reply:
Uhm, I don't think you've understood Markdown correctly. Best practices are to use `<>` at the beginning and end of a link. Because it's a Markdownfile, they're not being displayed with angle brackets, see https://github.com/AnonymousWP/handbook/blob/improve-docker-documentation/docs/devops-guide/docker.md#configuration.

Reply:
What's the point of making a sample URL which doesn't work clickable?

Reply:
It's not highly essential, but doesn't hurt either. It's to emphasize it's about a URL that you can access via your web browser when set.